x11-wm/i3/i3-4.22.ebuild

```diff
@@ -65,10 +66,6 @@ RDEPEND="
 "
 BDEPEND="virtual/pkgconfig"
 
-PATCHES=(
-	"${FILESDIR}"/${PN}-4.16-musl-GLOB_TILDE.patch
-)
-
```

Comment:
Are you sure about that? Also don't change 4.21.1, as said it is in a stablereq

Reply:
the GLOB_TILDE flag for the "glob" function has been supported in musl since 2019 from this commit https://git.musl-libc.org/cgit/musl/commit/?id=49eacf29d29fa704b2feda879978ae64a825358c . Before then, this patch was needed to implement the functionality manually.

Reply:
The comment in the patch file links to a different suggestion for the GLOB_TILDE functionality than the one in the patch. The patch pulls from HOME environment variable, musl does this too but also has a contingency for when the environment variable is not set. As far as I can see, that's the only functional difference.

Reply:
Good answer. Anyway I'd suggest to you not to touch that one as you are going to release a new version of i3. The ebuild can be dropped entirely when 4.22 will be stabled